### PR TITLE
db changes

### PR DESCRIPTION
**Pull request by priyanka-TL**

### CODE REVIEW

File: src/services/organization-extension.js [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+
+			if (bodyData.review_type === common.REVIEW_TYPE_SEQUENTIAL && bodyData.review_stages) {
+				//review stage creation
+				const reviewStages = bodyData.review_stages.map((stage) => ({
+					...stage,
+					organization_id,
+					resource_type,
+				}))
+
+				await reviewStageQueries.bulkCreate(reviewStages)
```
Comment (adithyadinesh0412) [inline src/services/organization-extension.js]: How are we getting the roles for review stages? @priyanka-TL

File: src/database/migrations/20240513082210-create-certificate-base-templates-table.js [hunk 2]
```diff
@@ -32,6 +33,14 @@ module.exports = {
 			meta: {
 				type: Sequelize.JSON,
 			},
+			created_by: {
+				type: Sequelize.STRING,
+				allowNull: true,
```
Comment (adithyadinesh0412) [inline src/database/migrations/20240513082210-create-certificate-base-templates-table.js]: Can we mandate created_by and updated_up?

File: src/database/migrations/20240514181708-add-all-permissions.js [hunk 3]
```diff
@@ -137,7 +137,7 @@ module.exports = {
 					code: 'review_permissions',
 					module: 'reviews',
 					request_type: ['POST', 'PATCH'],
-					api_path: '/scp/v1/reviews/update',
+					api_path: '/scp/v1/reviews/update*',
```
Comment (adithyadinesh0412) [inline src/database/migrations/20240514181708-add-all-permissions.js:140]: Should we add * for /scp/v1/projects/publish  api also?

File: src/database/queries/organizationExtensions.js [hunk 4]
```diff
@@ -1,15 +1,20 @@
...
-			throw error
-		}
+exports.create = async (data) => {
+	try {
+		return await OrgExtensions.create(data, { returning: true })
+	} catch (error) {
+		return error
+	}
+}
+
```
Comment (adithyadinesh0412) [inline src/database/queries/organizationExtensions.js:22]: @priyanka-TL
Please add findMany() back, its used in config listing api.  
```scp/v1/config/list```

File: src/database/migrations/20240513082210-create-certificate-base-templates-table.js (hunk 2 above)
Comment (priyanka-TL) [reply src/database/migrations/20240513082210-create-certificate-base-templates-table.js]: updated created_by as not nullable. but updated_by can be null

File: src/services/organization-extension.js (hunk 1 above)
Comment (priyanka-TL) [reply src/services/organization-extension.js]: from the req body.

Comment (adithyadinesh0412) [reply src/services/organization-extension.js]: I think it is not added in the api doc. can we add that? @priyanka-TL

File: src/database/migrations/20240513082210-create-certificate-base-templates-table.js (hunk 2 above)
Comment (adithyadinesh0412) [reply src/database/migrations/20240513082210-create-certificate-base-templates-table.js]: ok @priyanka-TL .

**Comment by adithyadinesh0412** [comment]
@priyanka-TL  Please check the merge conflicts also

**Comment by priyanka-TL** [comment]
> @priyanka-TL  Please check the merge conflicts also

Im working on the PR comments I will update once it is completed

**Comment by priyanka-TL** [comment]
@adithyadinesh0412 working on PR comments I will update once it is done

File: src/services/organization-extension.js (hunk 1 above)
Comment (priyanka-TL) [reply src/services/organization-extension.js]: @adithyadinesh0412 its added already

**Pull request merged**